tests/components/dwd_weather_warnings/test_init.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for Deutscher Wetterdienst (DWD) Weather Warnings integration."""
+
+from typing import Final
+
+from homeassistant.components.dwd_weather_warnings.const import (
+    CONF_REGION_IDENTIFIER,
+    DEFAULT_MONITORED_CONDITIONS,
+    DOMAIN,
+)
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import CONF_MONITORED_CONDITIONS, CONF_NAME
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+DEMO_CONFIG_ENTRY: Final = {
+    CONF_NAME: "Unit Test",
+    CONF_REGION_IDENTIFIER: "807111000",
+    CONF_MONITORED_CONDITIONS: DEFAULT_MONITORED_CONDITIONS,
+}
+
+
+async def test_config_entry_success(hass: HomeAssistant) -> None:
+    """Test setting up the integration from a config entry."""
+    entry = MockConfigEntry(domain=DOMAIN, data=DEMO_CONFIG_ENTRY)
+    entry.add_to_hass(hass)
+    await hass.config_entries.async_setup(entry.entry_id)
+
+    assert entry.state == ConfigEntryState.LOADED
+
+
+async def test_unload_entry(hass: HomeAssistant) -> None:
```

Comment:
if you move `assert entry.state == ConfigEntryState.LOADED` to this test you can call it `test_load_unload_entry` and skip the other test as it will test both loading and unloading.